src/components/text-editor/__internal__/editor-counter/editor-counter.style.ts

```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 
-const StyledCounter = styled.span`
+const StyledCounter = styled.span<{ hasError: boolean }>`
```

Comment:
It's used in the `editor-counter` component here:
``` <StyledCounter hasError={!!error}>{`${limit - count}`}</StyledCounter> ```

So I don't think it can be removed.

Reply:
sorry, not sure how on Earth I missed that. I'm going to resolve this and pretend I never said anything...